cmd/network-controller/main.go

```diff
@@ -66,6 +71,16 @@ func main() {
 
 	printVersion()
 
+	networkControllerConfig, err := ParseConfig(defaultConfigLocation)
+	if err != nil {
+		panic(err)
+		os.Exit(1)
```

Comment:
I am not sure, does the network-controller recover from panic? can this line actually be reached or is it dead?